doc/user-guide/04-scenarios.adoc

```diff
@@ -78,8 +78,14 @@ TAPE_DEVICE=/dev/nst0
 ----
 
 
-== Bootable ISO with Borg
+== ReaR with Borg back end
  - Install Borg backup (https://borgbackup.readthedocs.io/en/stable/installation.html).
+
+IMPORTANT:  We strongly recommend to use Borg standalone binary (https://github.com/borgbackup/borg/releases) as it includes all necessities for Borg operations.
```

Comment:
This documentation update (whole IMPORTANT block) addresses part of https://github.com/rear/rear/issues/1727